src/routes/safe/components/Transactions/TxList/styled.tsx

```diff
@@ -144,10 +144,14 @@ const gridColumns = {
 }
 
 const willBeReplaced = css`
+  .will-be-replaced {
```

Comment:
Have you checked how this affects other places this is used?

Reply:
I extended the use of this class to the Accordion Element of `TxQueueRow` so that it can not be opened anymore once in the disabled state. Other places where the class was used before like the text elements inside `TxDetails` should not be affected as they were non interactive to begin with.